deploy/gluster-block/kubernetes/csi-deployment.yaml

```diff
@@ -0,0 +1,295 @@
+# Copyright 2018 The Gluster CSI Authors.
```

Comment:
this template needs to  be updated to  use the latest sidecar CSI  images

Reply:
IMO create below directory struct

`example/kubernets/glusterfs/`   ---> for glsuterfs
`example/kubernetes/glsuter-loopback/`  ----> for loopback